### PR TITLE
Add readthedocs image

### PR DESCRIPTION
**Pull request by stefanDeveloper**

### CODE REVIEW

**Comment by stefanDeveloper** [comment]
Related to #27, however I had a problem updating the PR in time before it got merged :D

File: assets/heidgaf_logo_readthedocs.png
Comment (lamr02n) [inline assets/heidgaf_logo_readthedocs.png:1]: The logo seems too big in relation to the name. I think we should make it smaller.

That's how it looks on readthedocs:
image

**Review by lamr02n — CHANGES_REQUESTED** [review]
See comment on the readthedocs logo.

File: assets/heidgaf_logo_readthedocs.png
Comment (stefanDeveloper) [reply assets/heidgaf_logo_readthedocs.png:1]: Yes, I quickly adapted it without checking. Another thing is the view with dark mode.

Comment (stefanDeveloper) [reply assets/heidgaf_logo_readthedocs.png:1]: Screenshot 2024-09-27 at 09 00 36
Screenshot 2024-09-27 at 09 00 42

I adapted the size.

Comment (lamr02n) [reply assets/heidgaf_logo_readthedocs.png:1]: I like it!

**Review by lamr02n — APPROVED** [review]
Awesome, looks good!

**Pull request merged**